lib/pure/sugar.nim

```diff
@@ -160,16 +156,12 @@ macro dump*(x: untyped): untyped =
   ## It accepts any expression and prints a textual representation
   ## of the tree representing the expression - as it would appear in
   ## source code - together with the value of the expression.
-  ##
-  ## As an example,
-  ##
-  ## .. code-block:: nim
-  ##   let
-  ##     x = 10
-  ##     y = 20
-  ##   dump(x + y)
-  ##
-  ## will print ``x + y = 30``.
+  runnableExamples:
+    let
+      x = 10
+      y = 20
+    dump(x + y) # will print `x + y = 30`
```

Comment:
```suggestion
    if false: dump(x + y) # if true would print `x + y = 30`
```

Reply:
Why? Examples should be simple, not unneccessarily convoluted/wrapped.

Reply:
> Why?

https://github.com/nim-lang/Nim/pull/16802#discussion_r563438501